src/devices/CharacterLcd/samples/CharacterLcd.Samples.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
+    <ProjectReference Include="..\..\..\System.Device.Gpio\System.Device.Gpio.csproj" />
```

Comment:
This is why your build is failing right now. In order to get this working, you will need to use the same trick you did in the CharacterLcd project where you pass in the AdditionalProperties in order to set RuntimeIdentifier=Linux

Reply:
Whoops, missed this one